test/runtests.jl

```diff
@@ -0,0 +1,109 @@
+using PyPlot, Base.Test, PyCall
+
+#Checking if there's a GUI
+@test typeof(pygui()) == Symbol
+pygui(false)
+pygui(true)
```

Comment:
How do you specify that? I see a bunch of backend symbols [here](https://github.com/stevengj/PyCall.jl/blob/master/src/gui.jl#L46), but what's the exact call to run in non interactive mode?